search-app/src/cmr/search/api/autocomplete.clj

```diff
@@ -0,0 +1,77 @@
+(ns cmr.search.api.autocomplete
+  "Defines the API for autocomplete-suggest in the CMR."
+  (:require
+    [clojure.string :as string]
+    [compojure.core :refer :all]
+    [cmr.common.log :refer (debug info warn error)]
+    [cmr.common.mime-types :as mt]
+    [cmr.common.services.errors :as svc-errors]
+    [cmr.common-app.api.routes :as common-routes]
+    [cmr.search.api.core :as core-api]
+    [cmr.search.services.autocomplete-service :as ac]
+    [cmr.search.services.query-service :as qs]))
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;; Support Functions
+
+(def page-size-default 20)
```

Comment:
Why does autocomplete search have a different default page size at 20? why not make it 10 to be consistent with the other CMR endpoints?

Reply:
This is just the number that edsc suggested as a default

Reply:
EDSC is free to submit a page size with the autocomplete query. I don't think it is a good idea to have `default-page-size` defined in a bunch of places.

Reply:
I'll set it back to the default CMR of 10

Reply:
sorry i think there was a misunderstanding - can you use the `page-size-default` located in `cmr.common-app.services.search.query-model` so that there is one consistent source of truth?